Scrapper-Service/test/mongodb_test.py

```diff
@@ -0,0 +1,90 @@
+from datamodels import Conference
+from datamodels import Metadata
+from database  import mdb
+from datetime import datetime
+from datetime import timedelta
+from logging import Logger
+
+class MongoDbTestCase(unittest.TestCase):
+    def __init__(self, methodName):
+        super().__init__(methodName)
+        self.mongo_db = mdb.MongoDatabase(Logger,"Conference_Notify","conferences")
+    
+    def test_insert_datetime(self):
+        self.assertDictEqual.__self__.maxDiff = None
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            count = 0
+            conf = Conference(title="Something", 
+                                  url="www.something.com",
+                                  deadline=datetime.now(), 
+                                  metadata= Metadata("something", 
+                                                   datetime.now(), 
+                                                   "www.something.com", 
+                                                   "www.something.com",
+                                                   "something"
+                                                  ),
+                                  dateRange = [datetime.now(),datetime.now()+timedelta(days=10)],
+                                  finalDue = datetime.now(),
+                                  location = "something",
+                                  categories = ["something"],
+                                  bulkText = "somthing"
+                                ) 
+            
+            count = self.mongo_db.put(conf)
+            self.assertEqual(0,count)
+        
+    def test_valid_datetime(self):
+        self.assertDictEqual.__self__.maxDiff = None
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            count =0
+            conf = Conference(title="Something", 
+                                  url="www.something.com",
+                                  deadline=datetime.now()+timedelta(days=10), 
+                                  metadata= Metadata("something", 
+                                                   datetime.now(), 
+                                                   "www.something.com\somthing.html", 
+                                                   "www.something.com",
+                                                   "something"
+                                                  ),
+                                  dateRange = [datetime.now(),datetime.now()+timedelta(days=10)],
+                                  finalDue = datetime.now(),
+                                  location = "something",
+                                  categories = ["something"],
+                                  bulkText = "somthing"
+                                ) 
+            
+            count = self.mongo_db.put(conf)
+            self.assertEqual(1,count)
+            
+        def test_invalid_datetime(self):
```

Comment:
* Add comments+ meaningful names for function for others to know what these test will do.